stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/usecase/CancelMessage.kt

```diff
@@ -0,0 +1,31 @@
+package io.getstream.chat.android.offline.usecase
+
+import androidx.annotation.CheckResult
+import io.getstream.chat.android.client.call.Call
+import io.getstream.chat.android.client.call.CoroutineCall
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.livedata.utils.validateCid
+import io.getstream.chat.android.offline.ChatDomainImpl
+
+public interface CancelMessage {
```

Comment:
Use case are copy pasted from the live data package

Reply:
But why we need those copy and paste, instead of a move?